data/json/item_groups.json

```diff
@@ -9692,6 +9708,8 @@
       { "item": "rhubarb", "prob": 5, "count-min": 2, "count-max": 5 },
       { "item": "wild_herbs", "prob": 10 },
       { "item": "thyme", "prob": 5, "count-min": 1, "count-max": 2 },
+      { "item": "garlic", "prob": 1 },
+      { "item": "garlic_seed", "prob": 5, "count-min": 1, "count-max": 6 },
```

Comment:
I've found your ````garlic_seed```` problem; it's expecting ````garlic_seed```` in item_groups, but the actual item is ````seed_garlic````.